platform.md

```diff
@@ -146,94 +147,70 @@ The following is a non-exhaustive list of terms defined in the [OCI Distribution
 
 The platform SHOULD ensure that:
 
 - The image config's `User` field is set to a user with a **DIFFERENT** user [†](README.md#operating-system-conventions)UID/[‡](README.md#operating-system-conventions)SID as the build image.
-- The image config's `Label` field has the label `io.buildpacks.stack.maintainer` set to the name of the stack maintainer.
-- The image config's `Label` field has the label `io.buildpacks.stack.homepage` set to the homepage of the stack.
-- The image config's `Label` field has the label `io.buildpacks.stack.distro.name` set to the name of the stack's OS distro.
-- The image config's `Label` field has the label `io.buildpacks.stack.distro.version` set to the version of the stack's OS distro.
-- The image config's `Label` field has the label `io.buildpacks.stack.released` set to the release date of the stack.
-- The image config's `Label` field has the label `io.buildpacks.stack.description` set to the description of the stack.
-- The image config's `Label` field has the label `io.buildpacks.stack.metadata` set to additional metadata related to the stack.
+- The image config's `Label` field has the label `io.buildpacks.id` set to the target ID (e.g., "minimal") of the run image.
+- The image config's `Label` field has the label `io.buildpacks.base.maintainer` set to the name of the image maintainer.
+- The image config's `Label` field has the label `io.buildpacks.base.homepage` set to the homepage of the image.
+- The image config's `Label` field has the label `io.buildpacks.base.released` set to the release date of the image.
+- The image config's `Label` field has the label `io.buildpacks.base.description` set to the description of the image.
+- The image config's `Label` field has the label `io.buildpacks.base.metadata` set to additional metadata related to the image.
 
-### Mixins
+### Target Data
 
-A mixin name MUST only be defined by the author of its corresponding stack.
-A mixin name MUST always be used to specify the same set of changes.
-A mixin name MUST only contain a `:` character as part of an optional stage specifier.
+For both build images and run images, the platform MUST ensure that:
 
-A mixin prefixed with the `build:` stage specifier only affects the build image and does not need to be specified on the run image.
-A mixin prefixed with the `run:` stage specifier only affects the run image and does not need to be specified on the build image.
-A mixin WITHOUT a `build:` or `run:` prefix affects both the build and the run images.
+- The image config's `os` and `architecture` fields are set to valid identifiers as defined in the [OCI Image Specification](https://github.com/opencontainers/image-spec/blob/main/config.md).
+- The build image config and the run image config both specify the same `os`, `architecture`, `variant` (if specified), `io.buildpacks.distribution.name` (if specified), and `io.buildpacks.distribution.version` (if specified).
```

Comment:
```suggestion
- The build image config and the run image config both specify the same `os`, `architecture`, `variant` (if specified), `io.buildpacks.base.distro.name` (if specified), and `io.buildpacks.base.distro.version` (if specified).
```

Reply:
Is there a particular reason for changing `distribution` to `distro`? Would it not be better to remain consistent with e.g. `target.distribution`?

Reply:
Ah, do you mean the `[[targets]]` table in buildpack.toml? I guess there could be an argument for shortening that to `distro` as well. We already provide `CNB_TARGET_DISTRO_NAME` in the buildpack environment to indicate the target.

Here, we changed distribution -> distro because namespacing under `base` made the label very long.

Reply:
I'm :+1: for consistency if we're going to do that.